src/build.props

```diff
@@ -33,8 +33,9 @@
 
   <PropertyGroup Label="Package">
     <PackageId Condition=" '$(PackageId)' == '' ">$(AssemblyName)</PackageId>
+    <Title Condition=" '$(Title)' == '' ">$(AssemblyTitle)</Title>
     <Authors Condition=" '$(Authors)' == '' ">$(Company)</Authors>
-    <Owners Condition=" '$(Authors)' == '' ">$(Company),tse-securitytools</Owners>
```

Comment:
>Authors [](start = 27, length = 7)

This was wrong, and I fixed it, but that doesn't help -- see my comments in the PR description.